src/index.ts

```diff
@@ -134,21 +127,17 @@ const readAddress = (value: unknown): string => {
     throw new Error(`Invalid address value: ${value}`)
   }
 
-  if (value.length === 66) {
-    const EMPTY_SLOT = `0x${'0'.repeat(64)}`
-    if (value === EMPTY_SLOT) {
-      throw new Error('Empty slot')
-    }
-
-    return '0x' + value.slice(-40)
+  let address = value
+  if (address.length === 66) {
+    address = '0x' + address.slice(-40)
   }
 
-  const zeroAddress = `0x{'0'.repeat(40)}`
```

Comment:
This is the actual bug fix (missing `$`). The rest of the changes are just re-formatting (prettier).

Reply:
🙈 good catch